code/datums/gamemode/factions/bloodcult/Friendly Cult/friendlycult_runespells.dm

```diff
@@ -0,0 +1,230 @@
+#define RUNE_STAND	1
+
+/datum/rune_spell/friendly_cult
+	name = "Friendly Cult Spell"
+	desc = "How nice!"
+	invocation = "Lo'Rom Ip'Som"
+	
+	runeset_identifier = "friendly_cult"
+	
+/datum/rune_spell/friendly_cult/New(var/mob/user, var/obj/holder, var/use = "ritual", var/mob/target)
+	spell_holder = holder
+	activator = user
+	if(use == "ritual")
+		pre_cast()
+	
+/datum/rune_spell/friendly_cult/pre_cast() 
+	if(istype(spell_holder,/obj/effect/rune))
+		if((rune_flags & RUNE_STAND) && (activator.loc != spell_holder.loc))
+			abort(RITUALABORT_STAND)
+		else
+			invoke(activator,invocation)
+			cast()
+
+	
+	
+	
+/datum/rune_spell/friendly_cult/conjurecookie
+	name = "Conjure Cookie"
+	desc = "This simple rune summons a bowl of cookies filled with O+, Nar-Sie approved cookies."
```

Comment:
The cookies are o-, as they should be, not o+

Reply:
O-K